website/content/en/docs/main/tutorials/bookinfo/index.md

```diff
@@ -127,8 +127,8 @@ kubectl apply -f $BOOKINFO_FILES/review/rating.yaml
 ## Enable cross-cluster access
 
 In this step, we enable connectivity access for the BookInfo application
- by connecting the ProductPage service (client) to the reviews-v2 service (server1)
- and reviews-v3 (server2). We establish connections between the peers, export the reviews service on the server side,
+ by connecting the Product Page service (client) to the Reviews-V2 service (server1)
```

Comment:
we should be using the same service names as they appear in the application's deployment (not sure what they are, just asking for validation before changes). The bookinfo documentation probably has that.
I'm pretty sure the Service name for product page can't have a space in it.
This applies to other object name changes in this file as well

Reply:
I think it should be the same as the Istio documentation https://istio.io/latest/docs/examples/bookinfo/
They use the names:  ***reviews***, **productpage**, and **details**